charts/kubewarden-controller/values.yaml

```diff
@@ -162,17 +162,8 @@ preDeleteJob:
 # kubewarden-controller deployment settings:
 podAnnotations: {}
 nodeSelector: {}
-tls:
-  # source options:
-  # - "cert-manager-self-signed": Scaffold cert-manager integration, and create
-  #   a self-signed certificate with a cert-manager self-signed Issuer. Depends
-  #   on cert-manager. (default)
-  # - "cert-manager": Scafffold cert-manager integration. User configures their
-  #   own Issuer. Depends on cert-manager. Set tls.certManagerIssuerName to the
-  #   desired Issuer.
-  source: cert-manager-self-signed
-  # "cert-manager"-only options:
-  certManagerIssuerName: ""
+tolerations: []
+affinity: {}
```

Comment:
Are you sure about these new lines? I think these could be a leftover

Reply:
Yes, the affinity and tolerations are defined inside the global value now.

Reply:
sorry this is a conflict resolution that went bad, removing these lines